YoutubeDownloader/ViewModels/RootViewModel.cs

```diff
@@ -69,22 +71,28 @@ private async Task CheckForUpdatesAsync()
             if (updateVersion is null)
                 return;
 
-            Notifications.Enqueue($"Downloading update to {App.Name} v{updateVersion}...");
+            SnackbarHost.Post(new SnackbarModel($"Downloading update to {App.Name} v{updateVersion}...", _snackbarMessageDuration), null, DispatcherPriority.Normal);
             await _updateService.PrepareUpdateAsync(updateVersion);
 
-            Notifications.Enqueue(
+            SnackbarHost.Post(new SnackbarModel(
                 "Update has been downloaded and will be installed when you exit",
-                "INSTALL NOW", () =>
+                _snackbarMessageDuration,
+                new SnackbarButtonModel
                 {
-                    _updateService.FinalizeUpdate(true);
-                    RequestClose();
-                }
-            );
+                    Text = "INSTALL NOW",
+                    Action = () =>
+                    {
+                        _updateService.FinalizeUpdate(true);
+                        RequestClose();
+                    }
+                }),
+                null,
+                DispatcherPriority.Normal);
         }
         catch
         {
             // Failure to update shouldn't crash the application
-            Notifications.Enqueue("Failed to perform application update");
+            SnackbarHost.Post(new SnackbarModel("Failed to perform application update", _snackbarMessageDuration), null, DispatcherPriority.Normal);
```

Comment:
Maybe we should replace this call with an extension method, to avoid repeating `null, DispatcherPriority.Normal` everywhere? Assuming there is no simpler overload

Reply:
done, now the API is really similar as to before. I could have done it exactly the same, but I hope that the same SnackbarMessageQueue will be added in Material.Avalonia, and than it could conflict